src/submitter.py

```diff
@@ -273,6 +269,33 @@ def fetch_dumps(client, bucket, crash_id):
     return dumps
 
 
+COLLECTOR_KEYS = [
+    "collector_notes",
+    "dump_checksums",
+    "payload",
+    "payload_compressed",
+    "submitted_timestamp",
```

Comment:
Note that this shouldn't include "uuid" which is also added by the collector. The stage collector will see this uuid and reuse it. That allows crash reports to have the same uuid between stage and prod.